components/fxa-client/src/device.rs

```diff
@@ -106,14 +106,9 @@ impl FirefoxAccount {
     /// connected to the user's account. This includes applications that are registered as a device
     /// as well as server-side services that the user has connected.
     ///
-    /// This information is really only useful for targeted messaging or marketing purposes,
-    /// e.g. if the application wants to advertise a related product, but first wants to check
-    /// whether the user is already using that product.
-    ///
-    /// # Notes
-    ///
-    ///    - Attached client metadata is only visible to applications that have been
-    ///      granted the `https://identity.mozilla.com/apps/oldsync` scope.
+    /// It will only return active sessions.
+    /// For example, if a user has logged out of a service or session,
+    /// it wouldn't appear in this list.
```

Comment:
I don't want to derail this too much, but it's not clear logging out of a session actually removes this (ie, if I log in to relay in a browser and then log out, relay probably stays in the list). I *suspect* the only way to disconnect a service is via the "connected services" page on "manage account".

(Even then there seems a bug - for my account `listAttachedOauthClients` returned many duplicates for relay, and disconnecting relay from "manage account" removed most duplicates, but still left one for relay)

I'm not sure what point I'm making 😅 maybe to keep that 2nd para more vague, or say "user has disconnected the service from their account"?

(Regardless, thanks for caring and updating them!)

Reply:
Heh sorry a bit loose with logged out wording, but Mark is correct. This endpoint shows what we show in the connected devices section in settings. If a user has signed into a service and then signs out of it via settings, then this api would not return it.

<img width="1025" height="256" alt="Image" src="https://github.com/user-attachments/assets/0629bdf3-119e-427e-a832-65862920e3a9" />

Reply:
This is the documentation we use https://github.com/mozilla/fxa/blob/44e65172e68863cbb11761d0645b6f24ff5bc397/packages/fxa-auth-server/docs/swagger/devices-and-sessions-api.ts#L19, I'm updating it to call this out in https://github.com/mozilla/fxa/pull/19574.